joomla/joomla-cms
# [4.3] update help toc

Pull Request for Issue #40271 .
@obuisard Issue is a Release Blocker

### Summary of Changes

run `build/helpTOC.php`

New entries:

- [Guided Tours: New or Edit Step](https://docs.joomla.org/Help4.x:Guided_Tours:_New_or_Edit_Step)
- [Guided Tours: New or Edit Tour](https://docs.joomla.org/Help4.x:Guided_Tours:_New_or_Edit_Tour)
- [Guided Tours: Steps](https://docs.joomla.org/Help4.x:Guided_Tours:_Steps)
- [Guided Tours](https://docs.joomla.org/Help4.x:Guided_Tours)
- [Joomla Update](https://docs.joomla.org/Help4.x:Joomla_Update)

Renamed (because they are redirects):

| Old | New |
| --- | --- |
| [Articles: New or Edit Category](https://docs.joomla.org/Help4.x:Articles:_New_or_Edit_Category) | [Articles: Edit Category](https://docs.joomla.org/index.php?title=Help4.x:Articles:_Edit_Category&oldid=945626) |
| [Component: Field Groups](https://docs.joomla.org/Help4.x:Component:_Field_Groups) | [Field Groups](https://docs.joomla.org/index.php?title=Help4.x:Field_Groups&oldid=955524) |
| [Component: Fields](https://docs.joomla.org/Help4.x:Component:_Fields) | [Fields](https://docs.joomla.org/index.php?title=Help4.x:Fields&oldid=955286) |
| [Component: New or Edit Field](https://docs.joomla.org/Help4.x:Component:_New_or_Edit_Field) | [Fields: Edit](https://docs.joomla.org/index.php?title=Help4.x:Fields:_Edit&oldid=953058) |
| [Component: New or Edit Field Group](https://docs.joomla.org/Help4.x:Component:_New_or_Edit_Field_Group) | [Field Groups: Edit](https://docs.joomla.org/index.php?title=Help4.x:Field_Groups:_Edit&oldid=955690) |
| [Contacts: New or Edit](https://docs.joomla.org/Help4.x:Contacts:_New_or_Edit) | [Contacts: Edit](https://docs.joomla.org/index.php?title=Help4.x:Contacts:_Edit&oldid=945224) |
| [Contacts: New or Edit Category](https://docs.joomla.org/Help4.x:Contacts:_New_or_Edit_Category) | [Contacts: Edit Category](https://docs.joomla.org/index.php?title=Help4.x:Contacts:_Edit_Category&oldid=945854) |
| [News Feeds: New or Edit](https://docs.joomla.org/Help4.x:News_Feeds:_New_or_Edit) | [News Feeds: Edit](https://docs.joomla.org/index.php?title=Help4.x:News_Feeds:_Edit&oldid=946492) |
| [News Feeds: New or Edit Category](https://docs.joomla.org/Help4.x:News_Feeds:_New_or_Edit_Category) | [News Feeds: Edit Category](https://docs.joomla.org/index.php?title=Help4.x:News_Feeds:_Edit_Category&oldid=946688) |

### Testing Instructions

Check entries are present (administrator/index.php?option=com_admin&view=help)

~~Note: help links are still failing at the moment (see #39818)~~

### Actual result BEFORE applying this Pull Request

No links for `Guided Tours` in help site

![image](https://user-images.githubusercontent.com/66922325/236690956-99b40854-8888-4ad7-9fc0-7d87c7e8471e.png)

### Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/66922325/236690945-c53a0665-4221-4786-bec1-7c93f1af1923.png)

### Link to documentations
Please select:
